package/mdns-alias/Config.in

```diff
@@ -0,0 +1,7 @@
+config BR2_PACKAGE_MDNS_ALIAS
+	bool mdns-alias
+	select BR2_PACKAGE_DBUS
+	select BR2_PACKAGE_DBUS_PYTHON
+	help
+	  Advertises the initial $(HOSTNAME).local and network.local to
```

Comment:
Does it make sense for `netbrowse` to publish `network.local`? So that we won't risk blocking other devices in the network that are really running `netbrowse`. Or maybe rewrite `/etc/default/mdns-alias` when disabled.

Reply:
In the initial design, netbrowse publised both records, but after the refactor to a Flask app run by gunicorn, that's unfortunately no longer possible.  (Gunicorn calls only the app callbacks for each route, so there's no 'main' code running in the background.)

The best we can do right now is drop the alias when netbrowse is disabled.

Reply:
Rewrite of `/etc/default/mdns-alias` added, as well as a new `mdns-alias` daemon to replace the Python one.